src/storage/src/hummock/file_cache/metrics.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use prometheus::{
+    register_gauge_with_registry, register_histogram_with_registry,
+    register_int_counter_with_registry, Gauge, Histogram, IntCounter, Registry,
+};
+
+pub struct FileCacheMetrics {
+    pub disk_read_throughput: Gauge,
```

Comment:
IMO, using `Gauge` to calculate throughput can be inaccurate because it is a single value so it is possible for Prometheus to drop some data points and keep just the latest value. I think it is better to use a `Counter` to count read/write bytes and then use rate to calculate throughput in Grafana.